### PR TITLE
Api default locale codes WORKFLOWS-415

### PR DESCRIPTION
**Pull request by Varpusparvi**
Purpose:
Allow default_locale_codes parameter in member#update API as we have it in our frontend already.
Related changes:

https://github.com/phrase/phrase/pull/7300

### CODE REVIEW

**Comment by Varpusparvi** [comment]
Check it carefully, I'm pretty sure it's not correct. I'm not really at all sure if it should be plain string or [string].

File: paths/members/update.yaml [hunk 1]
```diff
@@ -34,21 +34,22 @@ x-code-samples:
...
-    --data '{"role":"Developer", "project_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235", "locale_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235", "permissions":"{"create_upload"=>true, "review_translations"=>true}"}' \
+    --data '{"role":"Developer", "project_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235", "locale_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235", "default_locale_codes":""de,en"", "permissions":"{"create_upload"=>true, "review_translations"=>true}"}' \
     --access_token <token>
 - lang: CLI v1
   source: |-
     phraseapp member update <account_id> <id> \
     --role Developer \
     --project-ids abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235 \
     --locale-ids abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235 \
+    --default_locale_codes "de,en" \
```
Comment (theSoenke) [inline paths/members/update.yaml:52]: We don't need this here as we do not update the old `phraseapp` CLI anymore

File: paths/members/update.yaml [hunk 2]
```diff
@@ -34,21 +34,22 @@ x-code-samples:
...
       -X PATCH \
-      -d '{"role":"Developer","project_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235","locale_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235","permissions":{"create_upload":true,"review_translations":true}}' \
+      -d '{"role":"Developer","project_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235","locale_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235","default_locale_codes":["de","en"],"permissions":{"create_upload":true,"review_translations":true}}' \
       -H 'Content-Type: application/json'
 - lang: CLI v2
   source: |-
     phrase members update \
     --account_id <account_id> \
     --id <id> \
-    --data '{"role":"Developer", "project_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235", "locale_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235", "permissions":"{"create_upload"=>true, "review_translations"=>true}"}' \
```
Comment (theSoenke) [inline paths/members/update.yaml:44]: This looks like too many quotes :)

File: paths/members/update.yaml [hunk 3]
```diff
@@ -34,21 +34,22 @@ x-code-samples:
...
+    --data '{"role":"Developer", "project_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235", "locale_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235", "default_locale_codes":""de,en"", "permissions":"{"create_upload"=>true, "review_translations"=>true}"}' \
```
Comment (forelabs) [inline paths/members/update.yaml:44]: I think we have to send it as array.

```suggestion
    --data '{"role":"Developer", "project_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235", "locale_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235", "default_locale_codes":["de","en"], "permissions":"{"create_upload"=>true, "review_translations"=>true}"}' \
```

**Review by forelabs — COMMENTED** [review]
looks good to me but since I'm not an openapi expert yet I will leave the approve to @theSoenke :slightly_smiling_face:

**Review by theSoenke — APPROVED** [review]
Looks good to me but I'm also not an OpenAPI expert. To check this change I would recommend building the client and trying out this new param

**Pull request merged**